ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -384,6 +384,49 @@
 {%             do port_channel_interfaces_isis.append(port_channel_interface) %}
 {%         endif %}
 {%     endfor %}
+{# VRRP #}
+{%     set ethernet_interfaces_vrrp_details = [] %}
+{%     for ethernet_interface in ethernet_interfaces | arista.avd.default([]) %}
+{%         if ethernet_interface.vrrp_ids is arista.avd.defined %}
+{%             do ethernet_interfaces_vrrp_details.append(ethernet_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if ethernet_interfaces_vrrp_details | length > 0 %}
+
+##### VRRP Details
+
+| Interface | VRRP-ID | Priority | Advertisement Interval | Preempt | Tracked Object Name(s) | Tracked Object Action(s) | IPv4 Virtual IP | IPv4 VRRP Version | IPv6 Virtual IP |
+| --------- | ------- | -------- | ---------------------- | --------| ---------------------- | ------------------------ | --------------- | ----------------- | --------------- |
+{%         for ethernet_interface in ethernet_interfaces_vrrp_details | arista.avd.natural_sort('name') %}
+{%             for vrid in ethernet_interface.vrrp_ids if vrid.id is arista.avd.defined %}
+{%                 set row_id = vrid.id %}
+{%                 set row_prio_level = vrid.priority_level | arista.avd.default('-') %}
+{%                 set row_ad_interval = vrid.advertisement.interval | arista.avd.default('-') %}
+{%                 set row_preempt = "Enabled" %}
+{%                 if vrid.preempt.enabled is arista.avd.defined(false) %}
+{%                     set row_preempt = "Disabled" %}
+{%                 endif %}
```

Comment:
Prempt is enabled by default on EOS. We do not configure `no vrrp x preempt` even when set to false. And we don't configure `vrrp x preempt` if set to true (without delay).
Delay and general preempt are two different commands in EOS:
```eos
 !
 interface Ethernet1/1
    shutdown
+   no vrrp 1 preempt
+   vrrp 1 preempt delay minimum 1 reload 2
 !
 interface Ethernet1/2
    shutdown
```

Reply:
that's the code from Vlan - should it be removed there as well?

Reply:
I think we should implement the rendering of `no vrrp x preempt` for the `false` case on both VLAN and Ethernet interfaces